promgen/templates/promgen/farm_detail.html

```diff
@@ -41,7 +41,7 @@ <h1>Farm: {{ farm.name }} ({{ farm.source }})</h1>
 
     <div class="panel-footer">
   {% if farm.editable %}
-      <a href="{% url 'hosts-add' farm.id %}" class="btn btn-primary">Add new host</a>
+      <a href="{% url 'hosts-add' farm.id %}" class="btn btn-primary">Register hosts</a>
```

Comment:
Maybe we should put this string in translation tags also `{% trans "Register hosts" %}`

Reply:
Done in [0c48714](https://github.com/line/promgen/pull/390/commits/0c487149b7869f980001ad57a53c7b8eb635cf1f).